src/mpi/comm/comm_split_type.c

```diff
@@ -177,6 +182,42 @@ static int node_split_processor(MPIR_Comm * comm_ptr, int key, hwloc_obj_type_t
   fn_fail:
     goto fn_exit;
 }
+
+static int node_split_io(MPIR_Comm * comm_ptr, int key, struct info_obj* info_obj_type,
+                                MPIR_Comm ** newcomm_ptr)
+{
+	int color;
+    hwloc_obj_t obj_containing_cpuset, io_device;
+    int mpi_errno = MPI_SUCCESS;
+
+    /* assign the node id as the color, initially */
+    MPID_Get_node_id(comm_ptr, comm_ptr->rank, &color);
+
+    obj_containing_cpuset =
+        hwloc_get_obj_covering_cpuset(MPIR_Process.topology, MPIR_Process.bindset);
+
+    if(info_obj_type->obj_type == HWLOC_OBJ_PCI_DEVICE){
```

Comment:
Please follow the mpich coding guidelines.  Space after `if`.

Reply:
Just run the code cleanup script. It'll take care of all of this.